pkgdb2client/utils.py

```diff
@@ -43,7 +43,7 @@
 def _get_bz(url=pkgdb2client.BZ_URL, insecure=False):
     ''' Return a bugzilla object. '''
     global BZCLIENT
-    if not BZCLIENT or BZCLIENT.url != url:
+    if not BZCLIENT:
```

Comment:
We already discussed this on IRC but I'll put it here for history.

I'm not fan of this change as it will break regenerating the bugzilla client object if a new URL is given. In the CLI context this is likely not such a problem but as pkgdb2client is also a library which can be used in script, I don't think I want to change this :)